distance_tolerance: 0.35
vitality_score:
  init: 2
  max: 6
validation_score_th: 4

base_frame_id: base_link
camera:
  frame_id: camera_center
  fov:
    x:
      min: 2.4
      max: 7.0
    bearing:
      min: -0.533
      max: 0.570
lidar:
  frame_id: lidar
  fov:
    x:
      min: 0.78
      max: 5.0
    y:
      min: -2.0
      max: 2.0

# data export
data_filename: fusion_data
map_frame: odom
